app/controllers/pieces_controller.rb

```diff
@@ -0,0 +1,13 @@
+class PiecesController < ApplicationController
+	def update
+    @piece = Piece.find(params[:id])
+    @game = @piece.game
+    @piece.move_to!(piece_params)
```

Comment:
You should wrap this in an `if` statement to ensure that the piece.user_id is equal to the current_user.id so that players cannot move pieces that aren't theirs.

Also, at the end you should likely redirect the user back to the game, so something like `redirect_to @game` as the last line of the method.